### PR TITLE
Update services.md

### PR DESCRIPTION
**Pull request by erxleben**
added some updates and information

### CODE REVIEW

File: freiburg/services.md [hunk 1]
```diff
@@ -17,47 +17,75 @@ The Freiburg Galaxy Team is offering several services to enable **reproducible**
...
+
+Locally in Freiburg, we offer twice per year a free full-week hands-on high-throughput sequencing data analysis workshop. A typical workshop schedule looks as follows: 
+
+- Monday:     Introduction to Galaxy and HTS
+- Tuesday:    Quality control, IGV, ChIP-Seq, 
+- Wednesday:  RNA-Seq 
+- Thursday:   HiC data analysis, Methyl-C data analysis
+- Friday:     Bring your own data, exercises
+
+Topice usually are selected from desired topics of the applicants.
```
Comment (bgruening) [inline freiburg/services.md]: Topics?

File: freiburg/services.md [hunk 2]
```diff
@@ -17,47 +17,75 @@ The Freiburg Galaxy Team is offering several services to enable **reproducible**
...
+
+All workshops are announced on our [events page](https://galaxyproject.eu/freiburg/events). Registrations are possible through our website.
+
+We and other members of the Galaxy training network (GTN) are giving training courses around the world for data analysis and for developers and administrators. Check the website of the main [Galaxy project](https://galaxyproject.org/events) as well as the [de.NBI](https://www.denbi.de) website for [de.NBI courses](https://www.denbi.de/training)
 
-In Freiburg, we already offer twice per year a full-week hands-on high-throughput sequencing data analysis workshop: Introduction to Galaxy and HTS, RNA-Seq, ChIP-Seq, Exome-Seq, MethylC-Seq, etc. Registrations are possible through our website.
+### Galaxy Training Material
 
-We are also giving training around the world on demand: for data analysis but also for developers and administrators.
+The Freiburg Galaxy Team is the main contributor to the community-driven development of GTN Galaxy training material. All of this material is **online, and freely accessible for everyone** at [https://training.galaxyproject.org](https://training.galaxyproject.org).
```
Comment (bgruening) [inline freiburg/services.md]: ""The Freiburg Galaxy Team is the main contributor" --> I think we should not mention ourself here that strong, maybe that pushes others back in contributing?

File: freiburg/services.md (hunk 1 above)
Comment (erxleben) [reply freiburg/services.md]: Take
"We have more training material for various topics to teach, e.g. Transcriptomics, Metabolomics, Metagenomics, Proteomics, Genome Annotation, Variant Calling, ... . Topics usually are selected from desired topics of the applicants.

File: freiburg/services.md (hunk 2 above)
Comment (erxleben) [reply freiburg/services.md]: is one of the main contributors.

Sorry, of course. This before was the old text

File: freiburg/services.md (hunk 1 above)
Comment (bgruening) [reply freiburg/services.md]: I mean there is a spelling mistake I think.

**Comment by bgruening** [comment]
Danke.

**Pull request merged**